modules/docker/login.go

```diff
@@ -0,0 +1,30 @@
+package docker
+
+import (
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+type LoginOptions struct {
+	Registry string
+	Login    string
+	Password string
+}
+
+// Login runs the 'docker login' command to login the given registry. This will fail the test if there are any errors.
+// Do not use production password, only for testing purpose.
+func Login(t testing.TestingT, opt LoginOptions) {
+	require.NoError(t, login(t, opt))
+}
+
+// login runs the 'docker login' command to login the given registry.
+func login(t testing.TestingT, opt LoginOptions) error {
```

Comment:
No public function which return error, similar to `Push() / PushE()`

Reply:
fix